lib/stat_tracker.rb

```diff
@@ -142,13 +142,31 @@ def worst_coach
   end
 
   def most_accurate_team
-
+    team_accuracies = calculate_team_accuracies
+    team_accuracies.max_by { |team, _| team_accuracies[team] }[0]
   end
 
   def least_accurate_team
+    team_accuracies = calculate_team_accuracies
+    team_accuracies.min_by { |team, _| team_accuracies[team] }[0]
+  end
+
```

Comment:
Nice use of helper method.

Reply:
I agree this makes the following code look clean